README.adoc

```diff
@@ -4,10 +4,6 @@
 The integration test repo contains a Stackable repository with services
 used for the integration tests.
 
-The following services are available:
```

Comment:
nit: I liked having the available services listed here, maybe we can just list them and link to the individual readmes?

Reply:
List of available services added